style.css

```diff
@@ -0,0 +1,38 @@
+head {
+    font-size: larger;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color:darkgreen
+}
+h1 {
+    font-size:60px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color:rgb(3, 61, 3)
+        
+}
+
+p {
+    font-size:25px;
+    font-family:'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color: rgb(3, 61, 3)
```

Comment:
If there are repeated style ruls across a few elements, try to think how you could abstract them so the styles are shared. This is better for maintainability

Reply:
e.g. these styles are shared in `h1` and `head` too